Sync.md

```diff
@@ -1,41 +1,129 @@
 ---
 uid: Sync
-title: Sync Types
+title: Download Options
 legacyUrl: /support/solutions/articles/44001162174-sync-types
 legacyUrl: /support/solutions/articles/44001162174-sync
 ---
 
-The sync feature allows you to copy content to external devices for backup purposes, archiving in multiple resolutions, and offline viewing.
+Emby client apps have a download icon and two buttons within the context menus, a **Download** button and a **Download to...** button. The latter gives the user the option to specify the target device, or if **Folder Sync** is configured and access to the user account is given, the target folder to copy media to.
```

Comment:
It's important to explain that

- The download button will be shown in all those client apps which support downloads themselves
- The download-to button is shown when the user has at least one client app which supports downloads

Reply:
I am re-writing this to be
```
Emby client apps have a download icon and two buttons within the context menus, a **Download** button and a **Download to...** button. The latter gives the user the option to specify the target device, or if [Folder Sync](Folder-Sync.md) is configured and access to the user account is given, the target server folder to copy media to.

User access rights and previously authenticated devices for the user, would determine what the available targets for the download are. 

* The **Download** button will be shown on all Emby client apps which support downloads. 

* The **Download to** button will be shown when the user has at least one client app which supports downloads, or if the user has been granted access to a [Folder Sync](Folder-Sync.md) server folder. 

In all cases, there are options to select the quality of the download plus other options depending on the media type. 

> [NOTE]
> The **Download** button on the [Emby Web app](Web-Client.md) is different and does not form part of the **Downloads & Sync** feature. The button, which is for single item downloads, uses the web browser functionality to download the item, without any conversion, to the default browser download folder. 
```